practice-three/db.json

```diff
@@ -0,0 +1,53 @@
+{
+  "userDatabase": [
```

Comment:
Just make it simple by using `users`. Follow https://restfulapi.net/resource-naming/